test/integration2/transaction/set-isolation-level.spec.js

```diff
@@ -10,6 +10,14 @@ describe('Transaction', () => {
         const tableName = 'key_value';
         before(() => {
           knex = getKnexForDb(db);
+
+          if (isMssql(knex)) {
```

Comment:
Do we still need the Docker config part then? I think it was originally added precisely because it was not clear how to do that on per-test basis.